src/main/java/com/SocialMedia/SocialMedia/models/dto/UserDTO.java

```diff
@@ -1,4 +1,22 @@
 package com.SocialMedia.SocialMedia.models.dto;
 
+import lombok.Getter;
+import lombok.Setter;
+
+import java.time.LocalDate;
+@Getter
+@Setter
+
```

Comment:
- @cristi6891 maybe we need to use @Data from Lombok (for All arguments and Null arguments Constructor, toString etc). 
- refactor code, remove free line (exem line 9 and line 21 etc)

Reply:
changed @Getter and @Setter with @Data
refactor done - removed free lines;

Reply:
ok